clusterloader2/pkg/measurement/common/probes/probes.go

```diff
@@ -56,13 +67,18 @@ func init() {
 	if err := measurement.Register(networkLatencyConfig.Name, create); err != nil {
 		klog.Errorf("cannot register %s: %v", networkLatencyConfig.Name, err)
 	}
+	create = func() measurement.Measurement { return createProber(dnsLookupConfig) }
+	if err := measurement.Register(dnsLookupConfig.Name, create); err != nil {
+		klog.Errorf("cannot register %s: %v", dnsLookupConfig.Name, err)
+	}
 }
 
 type proberConfig struct {
 	Name          string
 	MetricVersion string
 	Query         string
 	Manifests     string
+	Pods          []string
```

Comment:
This name is very misleading - you basically pass the value of labels of this pod, right?

Reply:
Changed to ProbeNameLabels.